saritasa_invocations/git.py

```diff
@@ -48,3 +50,248 @@ def clone_repo(
         printing.print_success(f"Pulling changes for {repo_link}...")
         with context.cd(repo_path):
             context.run("git pull")
+
+
+@invoke.task
+def blame_copy(
+    context: invoke.Context,
+    original_path: str,
+    destination_paths: str,
```

Comment:
Can't we pass destination paths as `*destination_paths`? I think it's more convenient to type it because of autocompletion

Reply:
invoke library doesn't provide such functionality. If we want to be able to handle list arguments we can do it through `@task(iterable="destination_paths")`, but then we will need to specify each element of `destination_paths` explicitly:
```
inv git.blame-copy original_file.py --destination_paths first_copy.py --destination_paths second_copy.py
```
https://docs.pyinvoke.org/en/stable/concepts/invoking-tasks.html#iterable-flag-values

Or we can use string argument separated by commas:
```
inv git.blame-copy original_file.py first_copy.py,second_copy.py
```